search/fibonacci_search.cpp

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright 2020 @author sprintyaf
```

Comment:
there is no need of a copyright all the code is already under MIT License, just write the author name.
```
@author sprintyaf
```

Reply:
the copyright issue is resolved